digdag-docs/src/operators.rst

```diff
@@ -997,3 +997,580 @@ Output Parameters
+
+  Examples:
+
+  * :command:`create_disposition: CREATE_IF_NEEDED`
+  * :command:`create_disposition: CREATE_NEVER`
+
+:command:`write_disposition: WRITE_TRUNCATE | WRITE_APPEND | WRITE_EMPTY`
+  Specifies whether to permit writing of data to an already existing destination table.
+
+  - ``WRITE_TRUNCATE``: If the destination table already exists, any data in it will be overwritten.
+  - ``WRITE_APPEND``: If the destination table already exists, any data in it will be appended to.
+  - ``WRITE_EMPTY``: *(default)* The query fails if the destination table already exists and is not empty.
+
+  Examples:
+
+  * :command:`write_disposition: WRITE_TRUNCATE`
+  * :command:`write_disposition: WRITE_APPEND`
+  * :command:`write_disposition: WRITE_EMPTY`
+
+:command:`priority: INTERACTIVE | BATCH`
+  Specifies the priority to use for this query. *Default*: ``INTERACTIVE``.
+
+:command:`use_query_cache: BOOLEAN`
+  Whether to use BigQuery query result caching.
+
+:command:`allow_large_results: BOOLEAN`
+  Whether to allow arbitrarily large result tables. Requires ``destination_table`` to be set and ``use_legacy_sql`` to be true.
+
+:command:`flatten_results: BOOLEAN`
+  Whether to flatten nested and repeated fields in the query results. *Default*: ``true``. Requires ``use_legacy_sql`` to be true.
+
+:command:`use_legacy_sql: BOOLEAN`
+  Whether to use legacy BigQuery SQL. Default: ``false``.
+
+:command:`maximum_billing_tier: INTEGER`
+  Limit the billing tier for this query.
+
+:command:`table_definitions: OBJECT`
+  Describes external data sources that are accessed in the query. For more information see `BigQuery documentation <https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.query.tableDefinitions>`_.
+
+:command:`user_defined_function_resources: LIST`
+  Describes user-defined function resources used in the query. For more information see `BigQuery documentation <https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.query.userDefinedFunctionResources>`_.
+
+
+Output parameters
+~~~~~~~~~~~~~~~~~
+
+:command:`bq.last_job_id`
+  The id of the BigQuery job that executed this query.
+
+
+bq_ddl>: Managing Google BigQuery Datasets and Tables
+-----------------------------------------------------
+
+The **bq_ddl>:** operator can be used to create, delete and clear Google BigQuery Datasets and Tables.
+
+
+.. code-block:: yaml
+
+    _export:
+      bq:
+        dataset: my_dataset
+
+    +prepare:
+      bq_ddl>:
+        create_datasets:
+          - my_dataset_${session_date_compact}
+        empty_datasets:
+          - my_dataset_${session_date_compact}
+        delete_datasets:
+          - my_dataset_${last_session_date_compact}
+        create_tables:
+          - my_table_${session_date_compact}
+        empty_tables:
+          - my_table_${session_date_compact}
+        delete_tables:
+          - my_table_${last_session_date_compact}
+
+
+Secrets
```

Comment:
Kept the `gcp.credential` instructions in the `bq>` operator section and linked the other bq operators to that.